src/utils/getCollections.ts

```diff
@@ -0,0 +1,37 @@
+import { paths, setParams } from '@reservoir0x/reservoir-sdk'
+import { Drop } from '@/config/partners/types'
+import { getNow } from '@/utils/getNow'
+
+export const getCollections = async (drops: Drop[]) => {
+  const now = getNow()
+  const contracts = drops.reduce((contracts, drop) => {
+    if (drop.address !== '0x' && drop.endDate && now >= drop.endDate) {
+      return [...contracts, drop.address]
+    }
+    return contracts
+  }, [] as `0x${string}`[])
+
+  if (contracts.length <= 0) {
+    return {}
+  }
+  const baseUrl = process.env.NEXT_PUBLIC_RESERVOIR_API_URL
+    ? process.env.NEXT_PUBLIC_RESERVOIR_API_URL
+    : 'https://api-base.reservoir.tools'
+  const url = new URL(`${baseUrl}/collections/v6`)
+  setParams(url, {
+    contract: contracts,
+  })
+
+  const response = await fetch(url.href)
```

Comment:
```suggestion
  const response = await fetch(url.href, { next: { revalidate: 0.5 * 60 } })
```

30s sounds good to me